packages/ui-components/src/assets/svg-symbols.svg

```diff
@@ -539,6 +539,15 @@
 		<path d="M6 2c-1.1 0-1.99.9-1.99 2L4 20c0 1.1.89 2 1.99 2H18c1.1 0 2-.9 2-2V8l-6-6H6zm8 6V2.5L19.5 8H14z"/>
 	</symbol>
 
+	<symbol id="kv-file-picker"
+		xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+		<path d="M18.21 5.615a2.25 2.25 0 0 1-2.25-2.25V.24H5.905a2.937 2.937 0 0 0-2.937 2.937V20.88a2.937 2.937 0 0 0 2.937 2.936h12.07a2.937 2.937 0 0 0 2.936-2.936V5.615z"/>
+		<path fill="#5B5B5B" fill-opacity=".3" d="M18.21 5.615a2.25 2.25 0 0 1-2.25-2.25V.24H5.905a2.937 2.937 0 0 0-2.937 2.937V20.88a2.937 2.937 0 0 0 2.937 2.936h12.07a2.937 2.937 0 0 0 2.936-2.936V5.615z"/>
```

Comment:
These fill attributes are probably unnecessary.

Do you think we can remove them?